src/components/mainLayout/mainHeader/mainHeader.tsx

```diff
@@ -30,6 +41,26 @@ const MainHeader = () => {
 		}
 	};
 
+	useEffect(() => {
+		setBaseUrl(getCorrectBaseUrl());
+	}, []);
+
+	useEffect(() => {
+		try {
+			if (baseUrl) {
+				validateToken(baseUrl).then(() => {
+					if (loginStatus) {
+						return router.push(UserProfilePath.ProfitMoney);
+					}
+				});
```

Comment:
validateToken(baseUrl).then((response: AxiosResponse<any, any>) => {
					if (loginStatus && response.status === HttpStatusCode.Ok) {
						return router.push(UserProfilePath.ProfitMoney);
					}
				});

Reply:
I have error with HttpStatusCode import. So I switched off eslint rules // eslint-disable-next-line import/named

Reply:
Just press button "Disable eslint rule for current line"